src/wallet/rpcwallet.cpp

```diff
@@ -929,11 +927,6 @@ static UniValue sendmany(const JSONRPCRequest& request)
 
     EnsureWalletIsUnlocked(pwallet);
 
-    // Check funds
-    if (totalAmount > pwallet->GetLegacyBalance(ISMINE_SPENDABLE, nMinDepth)) {
-        throw JSONRPCError(RPC_WALLET_INSUFFICIENT_FUNDS, "Wallet has insufficient funds");
```

Comment:
There was no test for this right?

Reply:
No, we are far away from test coverage in rpc/wallet.

However, there was (and still is) test coverage for the check a few lines down `RPC_WALLET_INSUFFICIENT_FUNDS`